packages/web-react/src/components/Button/index.ts

```diff
@@ -1,6 +1,12 @@
 export * from './Button';
-export * from './ButtonLink';
 export * from './useButtonAriaProps';
 export * from './useButtonStyleProps';
 export { default as Button } from './Button';
-export { default as ButtonLink } from './ButtonLink';
+
+// Export ButtonLink here for backwards compatibility
+// ButtonLink is NOT exported from packages/web-react/src/components/index.ts
+// ButtonLink will be removed in a future release and Button should be used instead
```

Comment:
Please, add also the link to the https://jira.lmc.cz/browse/DS-978

Reply:
link added